.env.example

```diff
@@ -0,0 +1,2 @@
+HOST=localhost
```

Comment:
Is this used for anything? AFAIK know `REACT_APP_*` envs are let through

Reply:
I have no idea? 🤷 Global search for `process.env.HOST` or similar should yield results if it's used.